xla/util_test.cc

```diff
@@ -134,6 +134,18 @@ TEST(UtilTest, RoundTripFpToString) {
   EXPECT_EQ(RoundTripFpToString(
                 -std::numeric_limits<tsl::float8_e4m3fn>::quiet_NaN()),
             "-nan");
+  EXPECT_EQ(RoundTripFpToString(
+                std::numeric_limits<tsl::float8_e4m3fnuz>::quiet_NaN()),
+            "-nan");
```

Comment:
You should use the string `"nan"` instead of `"-nan"`, otherwise this test case is basically the same as the one on the line below. The purpose is to test that both strings convert correctly to the single NaN value.

And same for the two `float8_e5m2fnuz` test cases below.

Also, you didn't cause this but it's a bit weird how the `float8_e4m3b11` test case on line 131 is between two `float8_e4m3fn` test cases. Since you're already modifying this function, can you move it below the second `float8_e4m3fn` case?

Reply:
The `fnuz` types (including `float8_e4m3b11fnuz`) only have a single NaN encoding (`0b1000000`), so they are always printed as `-nan` here because the sign bit is set. I'll simply delete the other test case, since it's redundant.

Reply:
Oh I was confused. I thought this was converting strings to floats, but it's really doing the opposite. Anyway this looks good now.